pilot/pkg/proxy/envoy/v2/ads.go

```diff
@@ -358,7 +358,13 @@ func receiveThread(con *XdsConnection, reqChannel chan *xdsapi.DiscoveryRequest,
 			totalXDSInternalErrors.Add(1)
 			return
 		}
-		reqChannel <- req
+		select {
+		case <-controlChannel:
+			adsLog.Errorf("ADS: %q %s stream has closed", con.PeerAddr, con.ConID)
```

Comment:
This function doesn't return any status, so I'd assume that even if it was an error, it could be logged by the code that caused the channel to be closed and here we'd simply return.

Reply:
agree ! simply return is enough